src/Umbraco.Core/Models/IDictionaryTranslation.cs

```diff
@@ -6,18 +6,24 @@ namespace Umbraco.Cms.Core.Models;
 public interface IDictionaryTranslation : IEntity, IRememberBeingDirty
 {
     /// <summary>
-    ///     Gets or sets the <see cref="Language" /> for the translation
+    ///     Gets or sets the <see cref="Language" /> for the translation.
     /// </summary>
-    [Obsolete("This will be removed in V13. From V13 onwards you should get languages by ISO code from ILanguageService.")]
+    [Obsolete("This will be removed in V14. From V14 onwards you should get languages by ISO code from ILanguageService.")]
     [DataMember]
     ILanguage? Language { get; set; }
 
-    [Obsolete("This will be replaced by language ISO code in V13.")]
+    [Obsolete("This will be replaced by language ISO code in V14.")]
     int LanguageId { get; }
 
     /// <summary>
-    ///     Gets or sets the translated text
+    ///     Gets or sets the translated text.
     /// </summary>
     [DataMember]
     string Value { get; set; }
+
+    /// <summary>
+    ///     Gets the ISO code of the language.
+    /// </summary>
+    [DataMember]
+    string LanguageIsoCode { get; }
```

Comment:
Unfortunately we can't add this member without a default Implementation, since that's a breaking change. I think we can get around this with something like: 

```C#
string LanguageIsoCode => Language?.IsoCode ?? string.Empty;
```